content/courses/intro-to-ada/chapters/tasking.rst

```diff
@@ -515,11 +524,11 @@ in a coordinated way, we use *protected objects*.
 Protected objects encapsulate data and provide access to that data by means
 of *protected operations*, which may be subprograms or protected
 entries. Using protected objects ensures that data is not corrupted by race
-conditions or other simultaneous access.
+conditions or other concurrent access.
 
 .. admonition:: Important
 
-    Protected objects can be implemented using Ada tasks. In fact, this was
+    Objects can be protected from concurrent access using Ada tasks. In fact, this was
     the *only* possible way of implementing them in Ada 83 (the first
```

Comment:
Because of the previous change, we should now replace "them" with "protected objects". Otherwise, it might get confusing.

Reply:
I don't believe we want to use the term "protected object" unless it is in fact an object whose type is a "protected type."  I intentionally used the term "object protected from concurrent access" to avoid the term "protected object" which has a very specific meaning in Ada.

Reply:
Actually, this admonition is in fact referring to protected objects. Maybe we could say:

"We can implement the same functionality of protected objects using just Ada tasks. In fact, this was [...]"

Reply:
I would suggest: "In fact, this was the only way of protecting objects from concurrent access in Ada 83 (the first ..."

Reply:
Perfect, I've just changed as suggested.